domain/src/main/java/org/oppia/domain/profile/ProfileManagementController.kt

```diff
@@ -22,9 +22,7 @@ import org.oppia.util.logging.Logger
 import org.oppia.util.profile.DirectoryManagementUtil
 import java.io.File
 import java.io.FileOutputStream
-import java.lang.Exception
-import java.util.Date
-import java.util.Locale
+import java.util.*
```

Comment:
Prefer importing each package rather than polluting the public namespace: https://developer.android.com/kotlin/style-guide#import_statements.

Reply:
Done.

Reply:
Yes Android studio is undoing it.